ampl-converter/src/test/resources/inputs/extended_exporter_v2/lcc-load-target-q-test-case.txt

```diff
@@ -0,0 +1,4 @@
+#LCC Converter Stations (hvdctest/InitialState)
+#"variant" "num" "bus" "con. bus" "substation" "q0 (MW)" "lossFactor (%PDC)" "powerFactor" "fault" "curative" "id" "description" "P (MW)" "Q (MVar)"
```

Comment:
There's a problem with the `q0` unit. It should be `"q0 (MVar)"`.
The same problem applies for the batteries (the same column name variable is used).

I propose to fix the batteries starting from this version of AMPL export. To do this:
- `AmplConstants.Q0` should be fixed;
- In `BasicAmplExporter.getBatteriesColumns()`, the following substitution should be applied:
```diff
- new Column(Q0),
+ new Column("q0 (MW)"), // Wrong unit, fixed in v1.2
```
- in `ExtendedAmplExporterV2`, the following method should be added:
```java
    @Override
    public List<Column> getBatteriesColumns() {
        List<Column> batteriesColumns = new ArrayList<>(super.getBatteriesColumns());
        batteriesColumns.set(BATTERY_Q0_COLUMN_INDEX, new Column(Q0)); // Fix the unit
        return batteriesColumns;
    }
```
- A unit test should be added for batteries in v1.2;
- `lcc-load-target-q-test-case.txt` should be updated (MVar instead of MW)

Reply:
With
```java
private static final int BATTERY_Q0_COLUMN_INDEX = 6;
```

Reply:
Thanks a lot for the careful review @olperr1, I'd missed it. The proposed changes have been implemented.